src/validation/schema.js

```diff
@@ -47,9 +47,14 @@ const typeSuggestion = typeName =>
     return typeof pattern === 'string' ? pattern === typeName : typeName.match(pattern)
   }).map(([_, suggestion]) => suggestion)[0]
 
-const loadSchema = filename => {
+const loadSchema = filenames => {
   try {
-    return fs.readFileSync(filename, 'utf-8')
+    // Concatenate multiple schema files together
+    let result = ""
+    for (filename of filenames) {
+      result += fs.readFileSync(filename, 'utf-8')
+    }
+    return result;
```

Comment:
Being the functional programmer that I am, I prefer this:
```js
return filenames
  .map(filename => fs.readFileSync(filename, 'utf-8'))
  .join('\n')
```

Reply:
But I'd rather stick to loading a single schema in `loadSchema`, instead of loading many files at once.